common/config/rush/common-versions.json

```diff
@@ -70,8 +70,6 @@
     "@azure/ms-rest-nodeauth": ["^0.9.2"],
     // Idenity is moving from v1 to v2. Moving all packages to v2 is going to take a bit of time, in the mean time we could use v2 on the perf-identity tests.
     "@azure/identity": ["2.0.0-beta.3", "^1.1.0"],
-    // App Config uses keyvault-secrets in a sample, switch to latest once the preview becomes GA
-    "@azure/keyvault-secrets": ["^4.1.0", "^4.2.0-beta.4"],
```

Comment:
@HarshaNalluru FYI - you ok with this change?

Reply:
This would break the app-config samples, right?
And we don't want the beta versions in our samples.

Reply:
No @azure/keyvault-secrets 4.2.0 GA'd today so figured we should use it https://www.npmjs.com/package/@azure/keyvault-secrets

Reply:
oh okay